c/src/ml-api-service-offloading.h

```diff
@@ -14,17 +14,61 @@
 #define __ML_SERVICE_OFFLOADING_INTERNAL_H__
 
 #include <ml-api-service.h>
+#include <nnstreamer-edge.h>
+
 #include "nnstreamer-tizen-internal.h"
 
 #ifdef __cplusplus
 extern "C" {
 #endif /* __cplusplus */
 typedef void * ml_service_offloading_h;
 
+/**
+ * @brief Enumeration for ml-offloading type.
+ */
+typedef enum
+{
+  ML_OFFLOADING_TYPE_UNKNOWN = 0,
+  ML_OFFLOADING_TYPE_TRAINING,
+
+  ML_OFFLOADING_TYPE_MAX
+} ml_offloading_type_e;
```

Comment:
The difference between `ml_offloading_type_e` and `ml_service_offloading_type_e`  is confusing.
Is it difficult to integrate these two struct?
If it is need to be separated, let's consider changing the name.
```
/**
 * @brief Enumeration for ml-offloading service type.
 */
typedef enum
{
  ML_SERVICE_OFFLOADING_TYPE_UNKNOWN = 0,
  ML_SERVICE_OFFLOADING_TYPE_MODEL_RAW,
  ML_SERVICE_OFFLOADING_TYPE_MODEL_URI,
  ML_SERVICE_OFFLOADING_TYPE_PIPELINE_RAW,
  ML_SERVICE_OFFLOADING_TYPE_PIPELINE_URI,
  ML_SERVICE_OFFLOADING_TYPE_REPLY,

  ML_SERVICE_OFFLOADING_TYPE_MAX
} ml_service_offloading_type_e;
```

Reply:
I will consider it, Could you see the last pr?